### PR TITLE
SPEC-318: Add upconversion implementation notes

### PR DESCRIPTION
**Pull request by durran**

### CODE REVIEW

**Comment by jyemin** [comment]
I'd also

**Comment by durran** [comment]
@jyemin Not sure why you closed it but I reopened and removed references to OP_COMMAND.

File: source/command-monitoring/command-monitoring.rst [hunk 1]
```diff
@@ -110,7 +110,47 @@ Bulk Writes
 
 This specification defines the monitoring of inidividual commands and in that repect MUST generate
 an event for each command a bulk write executes. Each of these commands, however, must be linked
-together via the same {{operationId}}.
+together via the same ``operationId``.
+
+Implementation Notes
+--------------------
+
+The ``command`` attribute of ``CommandStartedEvent`` MUST be upconverted to the corresponding
```
Comment (jyemin) [inline source/command-monitoring/command-monitoring.rst]: Re-reading this sentence, it's unclear what it means.  It's not the command attribute that is being up-converted, it's the legacy wire protocol messages.  This is clearer:

     Legacy wire protocol messages MUST be up-converted to the corresponding commands in order to....

**Comment by jyemin** [comment]
Fat-fingered and closed in error.  One more comment though.

**Comment by durran** [comment]
Changed

**Pull request merged**